### ISSUE TITLE
The CLI should not traceback when trying to save an already active override

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#1377]
I tried to use the CLI to alter the duration on an override I made today, and it crashes:

```
$ bodhi overrides save --duration 3 --notes "This is needed to build Bodhi." --user bowlofeggs python-pyramid-1.8.2-2.fc26
Password:
Traceback (most recent call last):
  File "/usr/bin/bodhi", line 11, in <module>
    load_entry_point('bodhi-client==2.4.0', 'console_scripts', 'bodhi')()
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/bodhi/client/__init__.py", line 229, in save_buildroot_overrides
    resp = client.save_override(nvr=nvr, duration=duration, notes=notes)
  File "/usr/lib/python2.7/site-packages/bodhi/client/bindings.py", line 78, in wrapper
    raise BodhiClientException(problems)
bodhi.client.bindings.BodhiClientException: Buildroot override for python-pyramid-1.8.2-2.fc26 already exists
```

It should be possible to use the CLI to edit active overrides, just like the web UI can do.

**Comment by ryanlerch** [comment]
+1 for removing the crash.

However, this highlights an issue with the naming of the `bodhi overrides save` command. It is only for *creating* a new override. There is the `bodhi overrides edit` command for editing an existing command.

**Comment by bowlofeggs** [comment]
On Wed, 2017-06-14 at 18:48 -0700, Ryan Lerch wrote:
> However, this highlights an issue with the naming of the bodhi
> overrides save command. It is only for creating a new override. There
> is the bodhi overrides edit command for editing an existing command.

Good point. I think the edit command was added after I filed this issue
(by @cverna). So yeah, probably we should add a "new" command and make
"save" be a deprecated alias for "new". And of course, we shouldn't
traceback, but maybe should just either do what the user wanted (if
it's just changing the expire date, why not do it for them?), or give a
useful error message.
